Lib/test/test_concurrent_futures.py

```diff
@@ -1312,6 +1317,15 @@ def test_shutdown_deadlock_pickle(self):
         # dangling threads
         executor_manager.join()
 
+    def test_crash_big_data(self):
+        self.executor.shutdown(wait=True)
```

Comment:
Can you add reference to the GH issue here?

Reply:
Done.